apps/crn-frontend/src/network/ProfileSwitch.tsx

```diff
@@ -0,0 +1,120 @@
+import { ComponentProps, FC, lazy } from 'react';
+import { useRouteMatch, Switch, Route, Redirect } from 'react-router-dom';
+
+import { NoEvents } from '@asap-hub/react-components';
+import { events } from '@asap-hub/routing';
+import { Frame, SearchFrame } from '@asap-hub/frontend-utils';
+
+import { EventConstraint } from '@asap-hub/model';
+
+const loadEventsList = () =>
+  import(/* webpackChunkName: "network-events" */ './EventsEmbedList');
+
+const EventsList = lazy(loadEventsList);
+
+type ProfileSwitchProps = {
+  About: FC;
+  Calendar?: FC;
+  currentTime: Date;
+  displayName: string;
+  eventConstraint: EventConstraint;
+  isActive?: boolean;
+  Outputs?: FC;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  route: any;
```

Comment:
apps/crn-frontend/src/hooks/current-user-profile-tab-route.ts Can we handle this with something like how we handled current profile route?